src/locales/en.json

```diff
@@ -28,6 +28,9 @@
     "edges": "Edges",
     "sharp": "Sharp",
     "round": "Round",
+    "decorators": "Decorators",
```

Comment:
Is "decorators" a term used in the wild? I've seen endings or caps.

Reply:
if we change the label I think we should also change the internal variable everywhere as well..

I vote for `endings` if not `decorators`

Reply:
I'm not sure where I picked it up from, possibly leaflet (https://github.com/bbecquet/Leaflet.PolylineDecorator). I'm open to alternatives though. Cap and join usually refer to the stroke shape, rather than some separate shape or symbol that uh decorates the start or end of the line. Different apps call them different names no absolute convention.

Sketch calls them "arrowheads". Of all the options, this feels the most excalidrawy to me.

Reply:
`arrow heads` is two words.. `caps` doesn't feel right and confusing, `endings` is straight forward.. and `decorators` is not bad :)

Reply:
> `arrow heads` is two words

Just one actually! (sorry 👨🏻‍🏫)

But given that we're adding shapes to both heads and tails, arrowheads might not be a good fit anyway. That's my only issue with endings, too: we'd have `endings.start` and `endings.end`. There's also the distant possibility of adding shapes to joins or in a rhythm along a line, which makes sense for some diagrams (see the Leaflet library above).

"Decorators" makes the most sense to me on an API level, because it leaves room for all of that, but it's an awful label in the UI. Endings or Arrowheads are better for the label. If we want to use the same term in both API and label then I would be happy with either term.

Reply:
> Just one actually! (sorry 👨🏻‍🏫)

On the properties window it will be called what? `Arrowhead` or `Arrow head`?

Reply:
You have a good point about keeping internally the term `decorator`, and I don't really mind having it decorators as a label as well. It's not the worse :)

Reply:
Yep, **Arrowheads**.

![image](https://user-images.githubusercontent.com/23072548/101292789-91635780-3809-11eb-94cb-8ac9fe3d1af4.png)

Reply:
head and tail if we want to be precise :)